content/docs/2.6/scalers/azure-app-insights.md

```diff
@@ -0,0 +1,197 @@
++++
+title = "Azure Application Insights"
+layout = "scaler"
+availability = "v2.6+"
+maintainer = "Community"
+description = "Scale applications based on Azure Application Insights metrics."
+go_file = "azure_app_insights_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-app-insights` trigger that scales based on an Azure Application Insights metric.
+
+```yaml
+triggers:
+- type: azure-app-insights
+  metadata:
+    metricAggregationTimespan: "0:1"
+    metricAggregationType: avg
+    metricFilter: cloud/roleName eq 'role_name'
+    metricId: "customMetrics/example-metric"
+    targetValue: "1"
+    activeDirectoryClientIdFromEnv: CLIENT_ID_ENV_NAME # Optional, can use TriggerAuthentication as well
+    activeDirectoryClientPasswordFromEnv: CLIENT_PASSWORD_ENV_NAME # Optional, can use TriggerAuthentication as well
+    applicationInsightsIdFromEnv: APP_ID # Optional, can use TriggerAuthentication as well
+    tenantIdFromEnv: TENANT_ID` # Optional, can use TriggerAuthentication as well
+```
+
+This scaler is backed by the Azure Application Instance REST API. Please see [this](https://docs.microsoft.com/en-us/rest/api/application-insights/metrics/get) page
+for further details.
+
+**Parameter list:**
```

Comment:
Sanity check - Are we listing all parameters here?

Reply:
A note has been added for pod identity. That should cover everything.